internal/worker/modelworkermanager/manifold.go

```diff
@@ -156,26 +161,26 @@ func (config ManifoldConfig) start(context context.Context, getter dependency.Ge
 	if err := getter.Get(config.StateName, &stTracker); err != nil {
 		return nil, errors.Trace(err)
 	}
-	statePool, systemState, err := stTracker.Use()
+	statePool, _, err := stTracker.Use()
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
```

Comment:
All of this can be deleted. Use the controller domain services instead to get a Model service. From there you can pass in the model uuid to get the model info where needed inside the modelworkermanager

You then delete shim.go entirely.

Reply:
Yup and refactored this, thanks ian!